### PR TITLE
DSG-727: fix simple choice questions

### PR DESCRIPTION
**Pull request by jgnuvalence**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

1. fixes the fact that when wanting to see the questions when editing a single-cjoise-question component, the questions were not being shown
2. remove support for images in single-choice-questions answers
3. The answers are left visible when opening the edit popub builder, this means that they no longer open in a different popup
4. the questions are added through the field label

## Screenshot of the affected changes

image
image

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts [hunk 1]
```diff
@@ -59,11 +57,10 @@ function editForm() {
               {
                 ...defaultDisplayBasicConfiguration,
                 components: [
-                  { ...defaultFieldLabelConfiguration },
+                  { ...cardRadioLegendConfiguration },
```
Comment (asultan001) [inline libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts:60]: Can you rename this const to be cardQuestionConfiguration instead?

File: libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts [hunk 2]
```diff
@@ -171,19 +150,11 @@ const simpleChoiceQuestions = {
       placeholder: 'Card title',
       type: 'textfield',
     },
-    {
```
Comment (asultan001) [inline libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts:174]: Did you mean to remove the content also? I think there is a difference between title and content and both could be useful here.

File: libs/shared/ui/nuverial/src/lib/components/radio-cards/radio-cards.component.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div>{{ legend }}</div>
 <br />
-<nuverial-card-group aria-labelledby="communication-preference" [formControl]="formControl" [required]="required">
```
Comment (davidyoung77) [inline libs/shared/ui/nuverial/src/lib/components/radio-cards/radio-cards.component.html:3]: Why is labelledby being removed here

File: libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts [hunk 4]
```diff
@@ -171,19 +150,11 @@ const simpleChoiceQuestions = {
...
-      type: 'textarea',
-    },
   ],
-  key: 'props.simpleChoiceQuestions',
+  key: 'props.answers',
   label: 'Answers',
   reorder: true,
   tableView: false,
-  tooltip: 'List of radio card options as key value pairs.',
+  tooltip: 'List of answer options as key value pairs.',
```
Comment (asultan001) [inline libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts:158]: These aren't key value pairs, I think you can just say List of answer options.

**Review by davidyoung77 — COMMENTED** [review]
Hey I have a question here, everything else looks good

File: libs/shared/ui/nuverial/src/lib/components/radio-cards/radio-cards.component.html (hunk 3 above)
Comment (jgnuvalence) [reply libs/shared/ui/nuverial/src/lib/components/radio-cards/radio-cards.component.html:3]: this was a mistake, David help me to fix a unit test that was failing when this attribute was present

File: libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts (hunk 4 above)
Comment (jgnuvalence) [reply libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts]: @asultan001 rhis is done

**Review by davidyoung77 — APPROVED** [review]
LGTM, approved pending @asultan001 comments

File: libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts (hunk 2 above)
Comment (jgnuvalence) [reply libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts]: I just put this back

File: libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts (hunk 1 above)
Comment (jgnuvalence) [reply libs/shared/feature/form-nuv/src/lib/components/simple-choice-questions/formio/formio-simple-choice-questions.model.ts]: @asultan001 rhis is done

**Pull request merged**